about.markdown

```diff
@@ -1,18 +1,73 @@
 ---
 layout: page
-title: About
+title: What is code curious
 permalink: /about/
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](https://jekyllrb.com/)
+## What is our mission?
 
-You can find the source code for Minima at GitHub:
-[jekyll][jekyll-organization] /
-[minima](https://github.com/jekyll/minima)
+<!-- Founded in April 2012, under our former name Rails Girls Berlin is the daughter of the **worldwide non-profit organization** [Rails Girls](http://railsgirls.com) that started in Finland. -->
+Formerly known as Rails Girls Berlin, we've been offering free programming workshops for women without any background in coding since 2012.
 
-You can find the source code for Jekyll at GitHub:
-[jekyll][jekyll-organization] /
-[jekyll](https://github.com/jekyll/jekyll)
+**We want to help to close the gender gap** in IT. Currently less than 20% of computer science majors are women, leaving women to be mostly consumers of technological products instead of creators and makers. Furthermore, it has been found that gender diversity leads to better decision making, increased creativity and enhanced, innovative performances.
```

Comment:
Is it really "only" the gender gap or is it `We thrive for more diversity in IT`?

Reply:
I would say, the focus is on the gender gap, because this is the most missrepresented ratio. Compared to a little more than 50% of women in all humanity, the ratio in tech is super wrong. While ratio for people with disability for example in all humanity is smaller and maybe in tech is not so much underrepresented compared to all humanity.

Reply:
Do you have data for this? 5% of companies must be disabled, yet many companies fail to reach this small number, instead paying fines.

I support the focus on the gender gap, because it's the one with the most data, but we should not exclude other marginalised and under-represented groups.

Reply:
There is data for this, I think we should research that properly anyways for future requests and presentations. I totally agree that we should not exclude other marginalized groups and often these things intersect anyways so that addressing one group alone will probably only work in theory. Let's reasearch and discuss here more. But historically we do come from the focus on women and used to still have this catalogue of underprivileged groups that would matter for a place in our workshops while being female or non cis male was kind of the base scope of our attendees.